### DISCUSSION TITLE
Loan Offers

### DISCUSSION THREAD

**Discussion by KrutikaMoorjani** [ONDC-Official/ONDC-FIS-Specifications#18] [Q&A] [answered]
**Sole Prop Flow**

Will there be an option to change loan tenor also on the buyer app? Figma mentions only request for lower loan amount

**Comment by 92shreyansh** [comment, marked as answer]
As part of the specification its present in 
Personal Loan:  https://github.com/ONDC-Official/ONDC-FIS-Specifications/blob/draft-2.x/api/components/examples/personal-loans/forms/personal-loans-loan-amount.html
Invoice Based Loan: https://github.com/ONDC-Official/ONDC-FIS-Specifications/blob/draft-2.x/api/components/examples/invoice-based-loans/forms/invoice-based-loans-set-loan-amount.html

field id: requestTerm
